index.js

```diff
@@ -0,0 +1,385 @@
+				res.render("dashboard_facility_table", {
+					res: req.session.user,
+					bookingData,
+					eventsData
+				});
+			});
+		});
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/request", (req, res) => {
+	if (req.session.user) {
+		res.render("request");
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/requestUpdate", (req, res) => {
+	if (req.session.user) {
+		con.query("select * from booking where id=4;", (err, bookingData) => {
+			con.query(
+				"select * from events where id=30;",
+				(err, eventsData) => {
+					res.render("requestUpdate", {
+						res: req.session.user,
+						bookingData,
+						eventsData
+					});
+				}
+			);
+		});
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/logout", (req, res) => {
+	res.clearCookie("session", {
+		path: "/"
+	});
+	res.redirect("/login");
+});
+app.post("/onLogin", (req, res) => {
+	con.query(
+		"select * from users where username= ?",
+		req.body.username,
+		(err, rows) => {
+			if (!err) {
+				if (rows.length > 0) {
+					if (
+						req.body.password ===
+						AES.decrypt(rows[0].password, `${process.env.AES_key}`)
+					) {
+						req.session.user = rows[0];
+						res.redirect("/dashboard");
+					} else {
+						console.log("Invalid password");
+						res.status(401).render("error", {
+							errorMessage: "Authentication error!"
+						});
+					}
+				} else {
+					console.log("Username not found!");
+					res.status(401).render("error", {
+						errorMessage: "Authentication error!"
+					});
+				}
+			} else {
+				console.log("Username not found!");
+				res.status(401).render("error", {
+					errorMessage: "Authentication error!"
+				});
+			}
+		}
+	);
+});
+app.post("/checkAvailability", (req, res) => {
+	if (req.session.user) {
+		con.query(
+			"SELECT slotHall FROM booking WHERE eventID IN (SELECT id FROM events WHERE dateWanted = ?)",
+			req.body.dateWanted,
+			(err, result) => {
+				if (!err) {
+					res.send(result);
+				} else {
+					console.log(err);
+					res.status(500).render("error", {
+						errorMessage: "Availability checking failed!"
+					});
+				}
+			}
+		);
+	} else {
+		res.redirect("/login");
+	}
+});
+app.post("/makeRequest", (req, res) => {
+	if (req.session.user) {
+		var slotHall = "";
+		for (var i = 1; i <= 7; i++)
+			for (var j = 1; j <= 9; j++) {
+				if (req.body[`cell${j}${i}`]) {
+					slotHall += j.toString() + i.toString();
+				}
+			}
+		var eventPayload = {
+			title: req.body.event_name,
+			club: req.body.clubName,
+			requirements: req.body.requirements,
+			details: req.body.desc,
+			dateWanted: req.body.dateWanted,
+			userID: req.session.user.id
+		};
+		con.query("insert into events set ?", eventPayload, err => {
+			if (!err) {
+				console.log(`Inserted ${eventPayload.title} into events.`);
+				con.query(
+					"select id from events WHERE userID= ? ORDER BY id DESC LIMIT 1",
+					req.session.user.id,
+					(err, eventID) => {
+						if (!err) {
+							var booking_obj = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'booking_obj' is not in camel case. (camelcase)](https://app.codacy.com/app/AravindVNair99/Hall-Management-System/pullRequest?prid=4592231)